client/index.jsx

```diff
@@ -1,9 +1,9 @@
-/* eslint-disable import/extensions */
-// /* eslint-disable import/no-extraneous-dependencies */
-import React from 'react';
-import ReactDOM from 'react-dom';
-import WebFont from 'webfontloader';
-import Reservations from './reservation.jsx';
+// /* eslint-disable import/extensions */
```

Comment:
Similar to others - comment as to why it's necessary to comment this out, why you need to keep it, etc.